code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -104,15 +104,27 @@
 
 		if("howl", "howls")
 			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
-			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
-			m_type = 2
+			if(miming)
+				message = "<B>[src]</B> tilts \his head back as if to howl."
+				m_type = 1
+			else
+				if(!muzzled)
```

Comment:
Unnecessary indentation here. This whole thing is equivalent to:
```
if(miming)
	...
else if(!muzzled)
	...
else
	...
```
PS. Why am I even here?